packages/styles/forms.css

```diff
@@ -193,6 +197,10 @@ textarea.Field--has-error:focus:hover,
   margin-bottom: var(--space-smallest);
 }
 
+.Checkbox .Error {
+  min-height: unset;
```

Comment:
Not directly related to this ticket, but the `Error` margin issue also applies to `Checkbox`.